assets/js/pages/ActivityLogPage/ActivityLogPage.jsx

```diff
@@ -139,6 +141,15 @@ function ActivityLogPage() {
               setSearchParams
             )}
           />
+          <Button
+            type="primary-white"
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** Maybe the button is too big? 
![image](https://github.com/user-attachments/assets/0a7edbc5-9b60-4af3-8928-d83546446b84)
‏
We may prefer something lighter like 
![image](https://github.com/user-attachments/assets/174f95a5-3e30-49ec-9c5a-9d86f2bbf9b9)


```suggestion
            type="link"
```

Reply:
Addressed like this 7cf3134e1d490c2fe582489c1a8f6b3afc64fdd7